### ISSUE TITLE
Windows airodump-ng - change dll load messages

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#196]
**Reported by darkAudax on 4 May 2007 13:35 UTC**

When airodump-ng is started under windows, it checks to see if the peek drivers have been installed.  The dialog box message is:

"Do you have installed peek drivers for your card?"

I would suggest the following dialog box message:

"The required peek drivers are not currently installed on your system.  Would you like to automatically download and install them now?"

If the user replies yes then proceed to download and install them.  If no, then exit the program.

As well, I noticed that a zero length byte file named "peek.sys" is created.  Is this required?  If no, then it should be eliminated.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 May 2007 13:35 UTC**

I don't really know how to make the program download and (re)install the right driver (I don't know which card he want to update and btw, there is not only one driver), so that part won't be done.

The 0 length file is normal, I create it once the user has clicked on "Yes they are installed", so that he doesn't get the popup every time it starts airodump-ng (on default windows install, such file is hidden).

To sum up, the change will be the following:
  * Message box text will stay unchanged
  * When the user click on "No", it will open a browser on the wildpacket website

**Comment by aircrack-ng** [comment]
**Modified by misterx on 4 May 2007 13:35 UTC**

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 May 2007 13:35 UTC**

Message will be:


```
Do you want the peek (from wildpackets) drivers installed?
Clicking on "No" assume you have it installed.
Clicking on "Yes" will open a browser on the driver download page.
```

Clicking on "Yes", airodump-ng will open the browser and then exit.

Clicking on "No", airodump-ng will simply continue.

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 May 2007 13:35 UTC**

(In #397) Changed DLL load message and open a browser if the user wish to download the peek driver (Closes: #196)

**Comment by aircrack-ng** [comment]
**Comment by anonymous on 4 May 2007 13:35 UTC**

Replying to [darkAudax](ticket:196):
> When airodump-ng is started under windows, it checks to see if the peek drivers have been installed.  The dialog box message is:
> 
> "Do you have installed peek drivers for your card?"
> 
> I would suggest the following dialog box message:
> 
> "The required peek drivers are not currently installed on your system.  Would you like to automatically download and install them now?"
> 
> If the user replies yes then proceed to download and install them.  If no, then exit the program.
> 
> As well, I noticed that a zero length byte file named "peek.sys" is created.  Is this required?  If no, then it should be eliminated.
>

**Comment by aircrack-ng** [comment]
**Comment by misterx on 4 May 2007 13:35 UTC**

Unless you have a patch, I'm not going to fix because this is way too old and not supported.
